docker/nginx/default.conf

```diff
@@ -1,13 +1,26 @@
 server {
-  listen 80 default_server;
+  listen 80 default_server reuseport;
```

Comment:
https://nginx.org/en/docs/http/ngx_http_core_module.html#reuseport

> Inappropriate use of this option may have its security [implications](https://man7.org/linux/man-pages/man7/socket.7.html).

I guess it is related to the port hijacking comment in `SO_REUSEPORT`?

Reply:
I've found this post: https://blog.cloudflare.com/the-sad-state-of-linux-socket-balancing/

While I think that we don't have "google scale", I see no harm to keep the option at the moment.